src/main/java/Deadline.java

```diff
@@ -0,0 +1,16 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private LocalDate deadline;
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-M-dd");
+    Deadline(String name, String deadLine) {
```

Comment:
Put a blank line to separate constructor from the private fields according to Java coding standard. And I think making the constructor public is better?